tests/layers/mapper/test_base_mapper.py

```diff
@@ -13,6 +13,10 @@
 
 
 class TestBaseMapper:
+    NUM_EDGES = 100
+    NUM_SRC_NODES = 100
```

Comment:
Since we changed the naming convention in anemoi-graphs, it should also be changed to `source` and `target` in these tests.